src/puma-cli.js

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env node
+'use strict';
+const program = require('commander');
+const puma = require('pumascript');
+const fs = require("fs");
+const injector = require('./injector');
+
+
+
+let run = (req, op) => {
+    puma.evalPuma(fs.readFileSync(req, 'Utf8'));
+}
+
+
+let compile = (req, op) => {
+    if (req.split('.')[1] !== 'puma') {
+        console.log('this file not is pumascript file');
```

Comment:
I do not understand this error. This is not a PumaScript file

Reply:
When the file does not have a .puma extension, I consider that the file does not have the pumascript code. That's why I'm sending that message through the console.